.github/workflows/test.yml

```diff
@@ -8,6 +8,9 @@ on:
     branches:
       - master
 
+permissions:
+  contents: read # to fetch code (actions/checkout)
```

Comment:
Above

Reply:
Yes, it makes sense. I thought since the workflow is small and one job only there may be a resistance to introduce two declarations :)